src/Sylius/Bundle/ApiBundle/Doctrine/Filter/ProductPriceOrderFilter.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Doctrine\Filter;
+
+use ApiPlatform\Core\Bridge\Doctrine\Common\Filter\OrderFilterInterface;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\AbstractContextAwareFilter;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use Doctrine\ORM\QueryBuilder;
+use Sylius\Component\Core\Model\ProductInterface;
+
+/** @experimental */
+final class ProductPriceOrderFilter extends AbstractContextAwareFilter
+{
+    protected function filterProperty(
+        string $property,
+        $value,
+        QueryBuilder $queryBuilder,
+        QueryNameGeneratorInterface $queryNameGenerator,
```

Comment:
This class should be used to generate parameter names like in [src/Sylius/Bundle/ApiBundle/Doctrine/Filter/ProductVariantOptionValueFilter.php:60](https://github.com/Sylius/Sylius/blob/e125bb158045185eb07fc4a862af53e06ec440fd/src/Sylius/Bundle/ApiBundle/Doctrine/Filter/ProductVariantOptionValueFilter.php#L60-L64)